src/Bicep.Core/FileSystem/PathHelper.cs

```diff
@@ -11,8 +11,12 @@ public static class PathHelper
 
         private const string TemplateOutputExtension = ".json";
 
+        private const string ParamsOutputExtension = ".bicepparam.json";
```

Comment:
Why not just use `.json`?

Reply:
Makes sense. I guess it would be confusing for the user to have to think about naming to avoid overlap. The most common convention I've seen for ARM JSON is to name the parameters `<something>.parameters.json`, so IMO might be worth renaming `.bicepparam.json` -> `.parameters.json`